markdown/dev/howtos/ways-to-contribute/technical-writing/patterns/en.md

```diff
@@ -2,6 +2,13 @@
 title: Writing for freesewing.org
 ---
 
-You could write documentation for freesewing.org, our makers website.
+A good way of contributing to FreeSewing is writting documentation! 
 
-You would need good writing skills and a familiarity with sewing.
+We don't have specific guidelines on how to write documentation. You can write as you feel fit as long as it's relevant and follows [our Code of Conduct](/guides/code-of-conduct/) and [our community standards](https://freesewing.org/docs/various/community-standards/).
+
+Once in a while you might see some documentation using personal tone, such as `Whilst a covered button is recommended you can use whatever button you desire, feel free to experiment!`. While the impersonal tone is more common for documentation, the personal tone shows that FreeSewing is not a souless entity, but instead we are a community of awesome people! 
+
+Whenever changing existing documentation (be it extending, correcting errors, etc) try to preserve the tone. Don't rewrite the documentation to solely change the tone.
+There may be cases, however, that documentation may need to extensively changed. In such occasions, it may be a good idea to get in touch with the author. 
```

Comment:
This is not particularly a comment about this paragraph, but we're at the *ways to contribute* pages here where we're looking to tell people that: *hey, there's almost certainly a way you too could contribute*.

This seems like a strange place to tell people *how* to contribute as we're in the process of trying to convince them to do so.